app/build.gradle

```diff
@@ -47,6 +47,7 @@ dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
     implementation 'androidx.appcompat:appcompat:1.3.0'
     implementation 'commons-io:commons-io:2.6'
+    implementation 'org.apache.commons:commons-text:1.9'
```

Comment:
I'm not sure whether pulling in this dependency is too heavyweight for the functionality that's used. Perhaps there's a more-lightweight method in the Java stdlib? Or potentially we can remove this more detailed logging entirely.

Reply:
SexyTopo app as distributed is tiny by any standards. We can handle a few K for a library.